### ISSUE TITLE
More Kids

### ISSUE THREAD

**Issue by moyaofthemist** [StardewModders/mod-ideas#251]
I would love to be able to have a big family on my farm instead of being limited to two kids. 

I’m not sure how feasible this is—maybe Robin could build an extra story on your house with lots of extra bedrooms (and a play room?) for the kids? Maybe to make the kids recognizable at a glance, the kids could be dressed in animal onesies/costumes like in Harvest Moon 64...? (that would also mean you wouldn’t have to worry about matching hair colors to parents, I guess, in addition to being adorable.)

Bonus: I’m not sure if Jax and Vincent’s playing sprites could be edited and repurposed to make it so your kids actually play up there, but it would be really wonderful if so. 

tl;dr: The specifics are less important to me than the basic concept—I just want a big, sprawling farm family.

**Comment by KarBoy2314PL** [comment]
there are multiple mods for no kids at all, but for more than 2? hmm

**Comment by minervamaga** [comment]
https://www.nexusmods.com/stardewvalley/mods/4515
